Assets/Scripts/Games/NipInTheBud/NipInTheBud.cs

```diff
@@ -116,42 +150,60 @@ public override void OnBeatPulse(double beat)
         {
             if (BeatIsInBopRegion(beat))
```

Comment:
bop regions are less flexible and less optimized (though it doesn't matter too much) than just making a list of beataction actions and making a new beataction with that